tests/unit/test_document.py

```diff
@@ -253,6 +253,41 @@ def test_get_entity_by_type(get_bytes_single_file_mock):
     assert actual[0].mention_text == "222 Main Street\nAnytown, USA"
 
 
+@mock.patch("google.cloud.documentai_toolbox.wrappers.document.storage")
```

Comment:
Can you pull/rebase from main, I think something got mixed up here. There should be another test_get_bytes() function

Reply:
right i removed it since the new test_get_bytes is doing the same thing as the other test i just added the DS_store stuff so it gets more coverage